component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/ComponentFamilyMeta.java

```diff
@@ -126,6 +135,26 @@ public ComponentBundle findBundle(final ClassLoader loader, final Locale locale)
                 }
             });
         }
+        
```

Comment:
This code was not modified to use the new model right? (The mojo as well btw)

Reply:
It wasn't. Work is in progress. Things to do:
1. Move getInput/OutputFlows methods from Meta to DesignModel (or Listener)
2. Change component-server to use it
3. Change mojo to use it
4. Check in Studio